src/usage/UsagePage.tsx

```diff
@@ -0,0 +1,87 @@
+// Libraries
+import React, {Dispatch, FC, useContext, useEffect, useReducer} from 'react'
+import {Page, RemoteDataState} from '@influxdata/clockface'
+
+// Components
+import PageSpinner from 'src/perf/components/PageSpinner'
+import UsageToday from 'src/usage/UsageToday'
+import RateLimitAlert from 'src/cloud/components/RateLimitAlert'
+import AlertStatusCancelled from 'src/billing/components/Usage/AlertStatusCancelled'
+import LimitChecker from 'src/cloud/components/LimitChecker'
+
+// Reducers
+import {
+  Action,
+  UsageState,
+  usageReducer,
```

Comment:
@Elders of the InfluxData. I was wondering if we should think about denoting types/interfaces as tSomething or iSomething? `usageReducer` and `UsageReducer` were confusing. Although, there's a difference in the casing. 

Thought I should ask.
@Asalem1 @drdelambre @121watts @genehynson

Reply:
Ya, definitely something to think about

Reply:
i think it's bike shedding, but if you feel strongly about it, figure out a way to automate it. We have a hard time enforcing things like this if we cant. There's been calls to alpha sort all of the imports, and you might occasionally find a // Libraries, // Components, // Types format for imports. And everyone keeps at it for awhile, and then a PR or two without it slip in, and then new people join and copy pasta from one of those "bad" files, and that rule isn't transferred amongst the million other things people are expected to know right off the bat....

if you can automate it, i support your crusade.

Reply:
https://en.wiktionary.org/wiki/bikeshedding

@drdelambre  Learned something. You have a valid point. Not sure if it's worth automating also.

Reply:
+1 on the bike shedding. Not worth the effort.